packages/contracts/test/utils/combinatorial_sets.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumber } from '@0xproject/utils';
+import * as _ from 'lodash';
+
+export const BigNumbers = [];
+
+const MAX_256_BIT_NUMBER = new BigNumber(2).pow(256).minus(1);
+
+export const positiveNaturalBigNumbers = [
+    new BigNumber(0),
+    new BigNumber(1),
+    new BigNumber(2),
+    new BigNumber(2).pow(64),
+    MAX_256_BIT_NUMBER,
+    MAX_256_BIT_NUMBER.div(2).floor(),
```

Comment:
This evaluates to `2**255-1`. This is the largest number that does *not* cause overflow when added to itself.

Is this what you intended?